main/ejml-experimental/benchmarks/src/org/ejml/sparse/BenchmarkRandomMatrixGenerator.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2009-2020, Peter Abeles. All Rights Reserved.
+ *
+ * This file is part of Efficient Java Matrix Library (EJML).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ejml.sparse;
+
+import org.ejml.sparse.csc.RandomMatrices_DSCC;
+
+import java.util.Random;
+
+public class BenchmarkRandomMatrixGenerator {
```

Comment:
Could you move this to `main/ejml-dsparse/benchmarks/src/org/ejml/sparse/csc` and JMH-ify it?

Reply:
Sure, I just couldn't find this directory before :)